ui/src/app/views/dataset-page.tsx

```diff
@@ -43,9 +45,17 @@ export const styles = reactStyles({
     fontSize: '16px',
     height: '2rem',
     lineHeight: '2rem',
-    paddingLeft: '13px',
+    paddingLeft: '0.55rem',
```

Comment:
what do these style changes do?

Reply:
>paddingLeft: '0.55rem',
- convert px to rem since rem is better to use than px when possible.
> paddingRight: '0.55rem',
- pad the right side the same amount as the left.

```
display: 'flex',
    justifyContent: 'space-between',
    flexDirection: 'row'
```
these attributes space elements so that they are as far apart as possible.

```
  selectBoxHeaderIconLinks: {
    fill: colors.blue[9]
  },	  },
```
defining the icon color in one location instead of 2 places